### PR TITLE
Start using Phenix

### PR DESCRIPTION
**Pull request by pschambacher**
:octopus::computer:

### Description
Use extraction of database spawn in a gem.

### Steps to merge
* [ ] :+1: from the team

### Risks
* None (test foo)

### CODE REVIEW

File: test/helper.rb [hunk 1]
```diff
@@ -12,28 +12,13 @@
 
 ActiveRecord::Base.logger = Logger.new(File.dirname(__FILE__) + "/test.log")
 
-config_content = IO.read(File.dirname(__FILE__) + '/database.yml')
-config_content = ERB.new(config_content).result
-config =  YAML.load(config_content)
-
-ActiveRecord::Base.configurations = config
+Phenix.configure do |config|
+  config.database_config_path = File.join(File.dirname(__FILE__), 'database.yml')
```
Comment (grosser) [inline test/helper.rb]: you said it's the default path !?

Comment (pschambacher) [inline test/helper.rb]: Works well for Gemfile but with the gemfiles/* Bundler.root is off

File: test/test_arhp.rb [hunk 2]
```diff
@@ -1,15 +1,20 @@
 require File.expand_path('helper', File.dirname(__FILE__))
 
 class ActiveRecordHostPoolTest < Minitest::Test
+  include Phenix
```
Comment (grosser) [inline test/test_arhp.rb]: prefer static methods over includes

File: test/helper.rb (hunk 1 above)
Comment (grosser) [inline test/helper.rb]: Dir.pwd ? :)

File: test/helper.rb [hunk 3]
```diff
@@ -12,28 +12,13 @@
...
+  config.schema_config_path   = File.join(File.dirname(__FILE__), 'schema.rb')
```
Comment (bquorning) [inline test/helper.rb]: Why not just `schema_path`? It’s not a config per se.

**Comment by grosser** [comment]
:+1:

**Pull request merged**